api/envoy/config/core/v3/health_check.proto

```diff
@@ -54,6 +54,11 @@ enum HealthStatus {
   DEGRADED = 5;
 }
 
+message HealthStatusSet {
+  // An order-independent set of health status.
+  repeated HealthStatus statuses = 1;
```

Comment:
Could this be an empty set? If not, then add a PGV constraint.

Reply:
It can be an empty set which will disable stateful session stickiness for current cluster.

Reply:
@snowp Yep. If override_host_status is set with an empty list of statuses then the override host will be ignored. And if override_host_status is not set completely, we will give some default statuses.